tools/sil-opt/SILOpt.cpp

```diff
@@ -230,6 +232,11 @@ static cl::opt<std::string> PassRemarksMissed(
              "the given regular expression"),
     cl::Hidden);
 
+static cl::opt<std::string>
+    RemarksFilename("sil-remarks-output",
```

Comment:
sil-opt options should match the spellings used by swiftc whenever there's a direct equivalent.